.gitignore

```diff
@@ -4,6 +4,8 @@ npm-debug.log
 Thumbs.db
 /coverage
 output.gltf
+CesiumTexturedBoxTest0FS.glsl
```

Comment:
I would rather ignore a single `output` directory that we commonly use for testing instead of the individual `output.gltf` and its resources.  Can you update this in #32 if you haven't already?